aws-iot-softwarepackageversion/contract-tests-artifacts/pre-script/buildspec.yml

```diff
@@ -0,0 +1,16 @@
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - echo Nothing to do in the pre_build phase...
+  build:
+    commands:
+      - echo Build started on `date`
+      - aws iot create-package --package-name CfnContractTestParentPackage --region $AWS_REGION
```

Comment:
Is there any way we could generate and random name and pass to cfn input? This line will fail when resource already exists